datafusion/core/Cargo.toml

```diff
@@ -79,6 +79,7 @@ object_store = "0.5.0"
 ordered-float = "3.0"
 parking_lot = "0.12"
 parquet = { version = "22.0.0", features = ["arrow", "async"] }
+parquet-format = "4.0.0"
```

Comment:
Should we move this logic to `arrow-rs` to avoid import this `parquet `

Reply:
I think that makes sense

Reply:
I think parquet `23.0.0` will have properly exposed arrow definitions (added by @tustvold  in https://github.com/apache/arrow-rs/pull/2626) 

Though now that I see this, I see the proposal is probably to move the functions `columns_sorted` and `check_is_ordered` into the `parquet` crate which makes sense to me (and will likely get reviewed by some others with more parquet knowledge)

@Ted-Jiang can you file a ticket / PR to do so?

Reply:
Sure!